packages/launcher/style/base.css

```diff
@@ -113,6 +113,8 @@
 .jp-LauncherCard:focus:not(:active) {
   border: 1px solid var(--jp-brand-color1);
   box-shadow: var(--jp-elevation-z6);
+  outline: unset; /* if outline is not unset, then depending on which browser you use, the
+    border change is either hidden behind the outline or visually combined with it */
```

Comment:
What is the rationale for adding the outline if we already have another visual indicator of focus?

Reply:
I'm not sure I follow. I'm removing the outline here. This was one of the places that I had to fix as a result of removing the bit of CSS that disables the outline:

```css
/* Disable default focus outline */
:focus {
  outline: unset;
  outline-offset: unset;
  -moz-outline-radius: unset;
}
```